lib/registries/addon/drafts/draft/draft-registration-manager.ts

```diff
@@ -87,6 +87,7 @@ export default class DraftRegistrationManager {
         const metadataValidations = buildMetadataValidations();
         const metadataChangeset = buildChangeset(draftRegistration, metadataValidations);
         set(this, 'metadataChangeset', metadataChangeset);
+        metadataChangeset.validate();
```

Comment:
Not sure we want to force validate here. In case, there aren't any changes (say new draft registration) this metadata will be marked as valid?? For page changesets, we only validate on init if the changeset already has responses.

Reply:
If there are no changes, then `metadataChangeset.validate()` will validate against the actual model instead of changes in the changeset.